iceoryx_posh/test/integrationtests/test_service_discovery.cpp

```diff
@@ -55,316 +107,314 @@ TEST_F(RoudiFindService_test, OfferSingleMethodServiceSingleInstance)
 }
 
 
-TEST_F(RoudiFindService_test, OfferServiceWithDefaultServiceDescriptionFails)
+TEST_F(ServiceDiscovery_test, OfferServiceWithDefaultServiceDescriptionFails)
 {
     ::testing::Test::RecordProperty("TEST_ID", "1db1ce50-4e95-46f3-8682-9cc90576dbc0");
-    auto isServiceOffered = senderRuntime->offerService(iox::capro::ServiceDescription());
+    auto isServiceOffered = sut.offerService(iox::capro::ServiceDescription());
     this->InterOpWait();
 
     ASSERT_EQ(false, isServiceOffered);
 }
 
-TEST_F(RoudiFindService_test, OfferServiceWithValidEventIdSucessfull)
+TEST_F(ServiceDiscovery_test, OfferServiceWithValidEventIdSucessfull)
 {
     ::testing::Test::RecordProperty("TEST_ID", "1107d0e3-42e1-4b24-9a4d-cef8badb7154");
-    auto isServiceOffered = senderRuntime->offerService({"service1", "instance1", "event1"});
+    auto isServiceOffered = sut.offerService({"service1", "instance1", "event1"});
     this->InterOpWait();
```

Comment:
No wait required IMO.

Reply:
I refactored the tests on a separate branch and the test you commented have already changed. I'll create a PR once this one is merged.